components/Calender/src/components/index.vue

```diff
@@ -148,8 +148,8 @@ function setQueryString(interations){
 
   if(interations) this.$router.replace(route)
 
-  const start = aDateTime.toUTC().toISO()
-  const end   = endDateTime.toUTC().toISO()
+  const start = aDateTime.toISO({includeOffset:false})
+  const end   = endDateTime.toISO({includeOffset:false})
```

Comment:
this option is to remove the offet indicator of ta date. 

I think it's an incorrect way that either fix/cause the entries that disapear on edge case (begining/end of week entries) we experience it in Nairobi that was ok in montreal `UTC-5` vs `UTC+2`. Clealy a timezone issue that the application does not seems to handle 

```javascript
{includeOffset:false}
```